### DISCUSSION TITLE
NovelLife?

### DISCUSSION THREAD

**Discussion by violavon** [Ralim/IronOS#1575] [General]
When looking Miniware products, I see NovelLife all the time. Does anyone know who they are? Just random clone manufacturer?

**Comment by DjordjeMandic** [comment]
Id say just a brand/store/reseller. Miniware/edesign still makes them. I bought MHP30 from NovelLife

**Reply by violavon** [reply]
Miniware's subsidiary, or other affiliation?

**Reply by DjordjeMandic** [reply]
i have no idea

**Comment by Ralim** [comment]
There are a lot of groups that just slap their brand on them and resell.
There are also a a few groups that make clones of the more popular ones; and then sell them OEM to these groups to slap their label on and resell.

Miniware only sells under (eDesign; Miniware; MiniDSO). All others are re-brands or clones done outside their control last I asked them around it.
